Algorithmia/algo_response.py

```diff
@@ -22,7 +22,7 @@ def create_algo_response(responseJson):
         # Parse response JSON
         if 'error' in responseJson:
             # Failure
-            raise parse_exception(responseJson['error'])
+            raise raiseAlgoApiError(responseJson)
```

Comment:
Looks like `raiseAlgoApiError` performs a `raise AlgorithmException(...)` itself, making this expr equivalent to `raise raise raise AlgorithmException(...)`. Calling the function without a `raise` should suffice I believe?

Reply:
Agreed, it doesn't break anything but functionally I think this is definitely more precise/reasonable. Updating.